ADOPTERS.md

```diff
@@ -0,0 +1,14 @@
+# Octant Adopters
```

Comment:
Adopters is a bit odd, mainly because you don't really adopt it .. this is like saying 'adopters of kubectl' ...

Reply:
It's meant to be seen as "users". We have the same terminology in other projects: https://github.com/vmware-tanzu/velero/blob/main/ADOPTERS.md